docs/lcgs/informal ebnf PRISM-inspired-2.txt

```diff
@@ -0,0 +1,76 @@
+start := (constdecl semicolon | simpledecl semicolon | moduledecl)*
+
+simpledecl := labeldecl
+    | vardecl
+    | playerdecl
+    | statevarchangedecl
+
+constdecl := "const" identifier eq expr;
+labeldecl := "label" identifier eq expr
+playerdecl := "player" identifier eq identifier relabeling?
+
+moduledecl := "template" identifier param? (simpledecl semicolon)* "endtemplate"
```

Comment:
transitiondecls are currently not allowed in the templates. Also, why not rename moduledecl to templatedecl.